src/main/java/org/jabref/gui/menus/FileHistoryMenu.java

```diff
@@ -33,6 +34,22 @@ public FileHistoryMenu(JabRefPreferences preferences, DialogService dialogServic
         }
     }
 
+    public boolean openFileByKey(KeyEvent keyEvent) {
+        if (keyEvent.getCharacter() == null) {
+            return false;
+        }
+        char key = keyEvent.getCharacter().charAt(0);
+        if (!Character.isDigit(key)) {
+            return false;
+        }
+        int num = Integer.parseInt(String.valueOf(key));
```

Comment:
I think you can directly use  getNumericValue of the char 
https://docs.oracle.com/javase/9/docs/api/java/lang/Character.html#getNumericValue-int-

Reply:
Yeah, this is a simpler way to do it.
I'll modify this. Thanks^_^